cmd/notary/tuf.go

```diff
@@ -249,6 +253,20 @@ func getTargetHashes(t *tufCommander) (data.Hashes, error) {
 	return targetHash, nil
 }
 
+// Open and read a file containing the targetCustom data
+func getTargetCustom(targetCustomFilename string) (*json.RawMessage, error) {
+	var targetCustom json.RawMessage
+	rawTargetCustom, err := ioutil.ReadFile(targetCustomFilename)
+	if err != nil {
+		return nil, err
+	}
+	err = json.Unmarshal(rawTargetCustom, &targetCustom)
```

Comment:
We should be doing `targetCustom.Unmarshal(rawTargetCustom)`. That will also require targetCustom to have been initialized, so line 258 needs to be `targetCustom := new(json.RawMessage)` then you'll also be able to remove the `&` when you reference it because `new` will already return a reference.